src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,40 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class FigureSupplier {
```

Comment:
https://github.com/mate-academy/jv-oop-advanced/blob/master/checklist.md#dont-use-static-methods-in-your-solution

1. Make all methods non-static
2. Public class members go to top, put private below